lib/OpenQA/Parser/Format/KTAP.pm

```diff
@@ -0,0 +1,171 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+package OpenQA::Parser::Format::KTAP;
+use Mojo::Base 'OpenQA::Parser::Format::Base', -signatures;
+
+# Translates KTAP kernel selftests format -> OpenQA internal representation
+use Carp qw(croak confess);
+use OpenQA::Parser::Result::OpenQA;
+use TAP::Parser;
+
+has [qw(test)];
+
+sub _testgroup_init ($self, $line, $test_ref, $steps_ref, $m_ref) {
```

Comment:
I am wondering if instead of having to pass all those references, they can be made members of `$self`

Reply:
huh really merged instantly once all checks are green.

I will not take it easy @perlpunk and will try it regardless.